docs/superpowers/specs/2026-07-23-app-navigation-changes-design.md

```
@@ -0,0 +1,201 @@
+
+**Discriminator (authoritative): how the current session was established.** A session established via PersonalID auth on an opportunity-linked app → Connect experience → resume Opp Home; a session established via manual login, or PersonalID auth on a non-opportunity-linked app → resume Standard Home. At implementation, verify the live-session signal agrees with `evaluateAppState`.
+
+**Inputs** (all already available):
+
+| Input | Source |
+|---|---|
+| Active user session alive? (~24h) | [`CommCareApplication.getSession().isActive()`](https://github.com/dimagi/commcare-android/blob/dc7697645fefd99de4e234be569bd8447fb6e0ba/app/src/org/commcare/CommCareApplication.java#L969) |
+| Seated app linkage/auth state | [`PersonalIdManager.evaluateAppState`](https://github.com/dimagi/commcare-android/blob/dc7697645fefd99de4e234be569bd8447fb6e0ba/app/src/org/commcare/connect/PersonalIdManager.java#L520) / [`ConnectJobHelper.getJobForSeatedApp`](https://github.com/dimagi/commcare-android/blob/dc7697645fefd99de4e234be569bd8447fb6e0ba/app/src/org/commcare/connect/ConnectJobHelper.kt#L23) |
+| PersonalID status | [`PersonalIdManager.isloggedIn()`](https://github.com/dimagi/commcare-android/blob/dc7697645fefd99de4e234be569bd8447fb6e0ba/app/src/org/commcare/connect/PersonalIdManager.java#L149) |
+| Connect access + opportunities | [`ConnectUserDatabaseUtil.hasConnectAccess()`](https://github.com/dimagi/commcare-android/blob/dc7697645fefd99de4e234be569bd8447fb6e0ba/app/src/org/commcare/connect/database/ConnectUserDatabaseUtil.java#L44) + opportunity records |
+| Installed apps present | [`MultipleAppsUtil.usableAppsPresent()`](https://github.com/dimagi/commcare-android/blob/dc7697645fefd99de4e234be569bd8447fb6e0ba/app/src/org/commcare/utils/MultipleAppsUtil.java#L50) || Last accessed opportunity | **new persistence — see below** |
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)